assets/css/_diversion_page.scss

```diff
@@ -0,0 +1,56 @@
+.l-diversion-page {
+  display: grid;
+  // Default layout is linear
+  grid-template:
+    "heading" min-content
+    "panel" auto
+    "map" 80vh
+    / 1fr;
+
+  &__header {
+    grid-area: heading;
+  }
+
+  &__panel {
+    grid-area: panel;
+  }
+
+  &__map {
+    grid-area: map;
+  }
+
+  @include media-breakpoint-up(md) {
+    // For larger breakpoints, layout into grid
+    grid-template:
+      "heading heading" min-content
+      "panel map" auto
+      / fit-content(28ch) auto;
+
+    // Overlay the panel and make the panel header match
+    // the size of the page header if subgrid is supported
+    @supports (grid-template-rows: subgrid) {
+      grid-template:
+        [panel-start] "heading heading" min-content
+        "panel map" auto
+        / fit-content(28ch) auto;
+
+      .c-diversion-panel,
+      .l-diversion-page__panel {
```

Comment:
I'm a little uncertain on the prefixes in this document, and would like input on picking a direction.

Because these are all layout classes pertaining to the Diversion _Page_, I opted to use the `l-`(layout) prefix for things directly within the Pages's control. Subcomponents, like the `c-diversion-panel`, use the `c-` prefix because it's targeting specific regions of a component. I could go either way on if `c-diversion-panel` should be a `c-`(component) prefix or a `l-`(layout) prefix.

Reply:
I think I like this current breakdown of the use of `c-` versus `l-`.